website/www/site/content/en/contribute/_index.md

```diff
@@ -83,217 +76,136 @@ sudo apt-get install \
    docker-ce
 ```
 
-On some systems (like Ubuntu 20.04) these need to be installed also
-```
-pip3 install grpcio-tools mypy-protobuf
-```
```

Comment:
We shouldn't drop this either.

Reply:
I will change all contents manually at monday and tuesday, and I will try to remember to keep it.